components/Cards/src/MDCCard.m

```diff
@@ -317,4 +328,12 @@ - (void)updateBackgroundColor {
   self.layer.shapedBackgroundColor = _backgroundColor;
 }
 
+- (void)setInteractable:(BOOL)interactable {
```

Comment:
nit: Auto-synthesize and remove these methods?
Here and in the cell implementation below.